test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -591,132 +573,7 @@ public void AddPicture_should_not_change_the_underlying_file_size()
         var addedPictureImage = shapes.Last<IPicture>().Image!;
         addedPictureImage.AsByteArray().Length.Should().BeLessThan(38000);
     }
-
-    [Test]
-    public void AddPicture_should_not_duplicate_the_image_source_When_the_same_png_image_is_added_twice()
```

Comment:
Please, do not remove this and another below `AddPicture_***` tests

Reply:
This looks to have happened in the merge conflict commit you did. Do you want me just to revert that?

Reply:
Yes, try to make revert to keep these tests or maybe it would be easier to just copy past these tests from origin `master`.

Reply:
The diff is borderline unintelligible but I think this is sorted now?

Reply:
I've pushed some changes in your branch `Add-SVG-Support-For-Mac-and-Linux`. Please pull.